lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -288,6 +288,25 @@ abstract class ViewPadding {
   }
 }
 
+abstract class ViewConstraints {
```

Comment:
I wonder if we can just show the same class coming from the normal `ui` package.

Reply:
I modeled this after ViewPadding above. But, yeah, I was also surprised that all this is repeated in web_ui...